frontend/src/components/dashboard/PremiumPromoBanners.tsx

```diff
@@ -0,0 +1,131 @@
+import { useLocalization } from "@fluent/react";
+import { ReactNode } from "react";
+import styles from "./ProfileBanners.module.scss";
+import PhoneIllustration from "./images/phone-premium-promo.svg";
+import { RuntimeData } from "../../../src/hooks/api/runtimeData";
+import { Banner } from "../Banner";
+import { isPeriodicalPremiumAvailableInCountry } from "../../functions/getPlan";
+import { ProfileData } from "../../hooks/api/profile";
+
+export type Props = {
+  profile: ProfileData;
+  runtimeData?: RuntimeData;
+};
+
+export const PremiumPromoBanners = (props: Props) => {
+  const banners: ReactNode[] = [];
+
+  if (
+    !props.profile.has_premium &&
+    isPeriodicalPremiumAvailableInCountry(props.runtimeData)
```

Comment:
Suggestion: Can we add a waffle flag to hide these by default? I don't want this to ship with all four of these displayed accidentally. 

I do realize that we may need to ship one of these banners. If that's necessary, check with Mel/Eduardo on which is the preferred one to ship.

Reply:
Good idea - I'll ask the rest of the team.